tools/dashboard/dashboard.conf

```diff
@@ -202,6 +202,18 @@ report_url:  https://storage.googleapis.com/cp2k-ci/dashboard_hip-rocm-build_rep
 
 # ==============================================================================
 
+[intel-oneapi-hpckit-ssmp]
```

Comment:
The `dashboard.conf` is only for displaying test results.
To actually run the new tests, you'll also need to add them to [tools/docker/cp2k-ci.conf](https://github.com/cp2k/cp2k/blob/master/tools/docker/cp2k-ci.conf).